code/modules/antagonists/wizard/equipment/spellbook.dm

```diff
@@ -501,6 +501,34 @@
 	playsound(get_turf(user), 'sound/effects/ghost2.ogg', 50, 1)
 	return TRUE
 
+/datum/spellbook_entry/summon/portals
+	name = "Summon Portal Storm"
+	desc = "Terrorize the crew with a portal storm! Whatever crawls out of these portals will be a threat not just to the crew, but you as well!"
+	cost = 3
+
+/datum/spellbook_entry/summon/portals/IsAvailible()
+	if(!SSticker.mode)
+		return FALSE
+	else
+		return TRUE
+
+/datum/spellbook_entry/summon/portals/Buy(mob/living/carbon/human/user, obj/item/spellbook/book)
+    SSblackbox.record_feedback("tally", "wizard_spell_learned", 1, name)
+    var/portaltype = pick(50;"port_shocktroop",50;"port_narsiespawn")
```

Comment:
```suggestion
    var/portaltype = pick("port_shocktroop","port_narsiespawn")
```

NO WEIRD SYNTAX HERE

If you want to add more storms with altering chances if this PR gets merged then do so when you are making the new storms. [YAGNI](http://wiki.c2.com/?YouArentGonnaNeedIt).